tower-reconnect/src/future.rs

```diff
@@ -1,25 +1,32 @@
 use crate::Error;
-use futures::{Future, Poll};
+use std::{
+    future::Future,
+    pin::Pin,
+    task::{Context, Poll},
+};
 
 pub struct ResponseFuture<F> {
     inner: F,
 }
 
+impl<F> Unpin for ResponseFuture<F> {}
```

Comment:
The argument here is that we never move `inner` and never give out `&mut` to it?